### PR TITLE
make the bundle compatible with Symfony 3.0

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

**Comment by xabbuh** [comment]
@mogilvie I reverted the changes to the `addFlash()` method.

File: Form/Type/CronType.php [hunk 1]
```diff
@@ -4,12 +4,19 @@
...
 use \Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
 use Symfony\Component\OptionsResolver\OptionsResolverInterface;
 
 class CronType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $textType = 'Symfony\Component\Form\Extension\Core\Type\TextType';
```
Comment (xabbuh) [inline Form/Type/CronType.php]: @mogilvie There was a typo in FQCN here before. Can you please try it with the current version?

**Comment by mogilvie** [comment]
@xabbuh all seems in order and working.

**Comment by xabbuh** [comment]
@mogilvie Thanks for the confirmation.

**Comment by NicolasBadey** [comment]
If everything is good for you I will create a v3.0 tag

**Pull request merged**